pcsx2/PAD/Linux/SDL/joystick.cpp

```diff
@@ -82,12 +82,23 @@ void JoystickInfo::Rumble(unsigned type, unsigned pad)
 		return;
 
 	if (m_haptic == nullptr)
-		return;
+	{
+		// If haptic is not implemented for a controller, try to use rumble API instead
+		static const uint16_t rumble_force = SDL_MAX_UINT16;
+		static const uint16_t big_motor[NB_EFFECT] = { 0, rumble_force, };
+		static const uint16_t small_motor[NB_EFFECT] = { rumble_force, 0, };
 
-	int id = m_effects_id[type];
-	if (SDL_HapticRunEffect(m_haptic, id, 1) != 0)
+		if (SDL_GameControllerRumble(m_controller, big_motor[type], small_motor[type], s_effect_duration_ms) != 0)
+		{
+			Console.Error("ERROR: Rumble is not working! %s, type is %u\n", SDL_GetError(), type);
```

Comment:
You can remove \n at the end since it's done by the console already in this case. Same for others.

Reply:
Done